docs/purpose.md

```diff
@@ -0,0 +1,15 @@
+## Purpose
+
+Today, the primary purpose of Online Boutique is to demonstrate:
+
+* [Google Kubernetes Engine (GKE)](https://cloud.google.com/kubernetes-engine)
+* [Anthos](https://cloud.google.com/anthos)
+* [Google Cloud Operations](https://cloud.google.com/products/operations)
+* tools and technologies commonly used alongside the above products
```

Comment:
```suggestion
* [Google Kubernetes Engine (GKE)](https://cloud.google.com/kubernetes-engine).
* [Anthos](https://cloud.google.com/anthos).
* [Google Cloud Operations](https://cloud.google.com/products/operations).
* Tools and technologies commonly used alongside the above products.
```

Reply:
This sentence ends with:
> while being accessible and useful to all new Kubernetes users.
So I'll will pass on this suggestion.
Thanks, Olivier.